.github/workflows/build.yml

```diff
@@ -1,5 +1,10 @@
-name: Build and Deploy
-on: [push, pull_request]
+name: Test and Deploy
+on:
+  pull_request:
+    types: [assigned, opened, reopened, synchronize]
+  push:
+    branches:
+      - master
```

Comment:
Can we somehow test on all branches but deploy only on `master`?

Reply:
I hope this is what this does, will have to test it by merging...